manager/usermanager/createUser.go

```diff
@@ -38,8 +35,26 @@ func CreateUser(userStore *store.UserStore, user *models.UserCredentials) (*mode
 			UnverifiedEmail: user.UserName,
```

Comment:
unverified_email is set to username, is it because username is the unique key the end user uses to sign-up? Or should it be `UnverifiedEmail: user.UnverifiedEmail` like the else one?

Reply:
ACtually as explained in this comment https://github.com/mayadata-io/kubera-auth/pull/51#discussion_r596696442. We take `UnverifiedEmail` from the UI and store it under `Username` field as for signup these both are going to be one and the same, but if we store it under `unverified_email` we will face problem while checking that a user with the same username is already existing or not .